api/src/main/java/openconsensus/trace/ImmutableEvent.java

```diff
@@ -30,7 +31,7 @@
  */
 @Immutable
 @AutoValue
-public abstract class Event {
+public abstract class ImmutableEvent extends Event {
```

Comment:
can this class be hidden and we will provide static `create` method on `Event`? It feels unnecessary to create another type of event.

Reply:
Done. Let's address the interface change separately.